src/resources/style.css

```diff
@@ -122,7 +122,7 @@ a.active:hover { text-decoration: none; }
 /* run console ansi colors (based on base16-default-dark and base16-bright) */
 :root {
   --ansi-black: #181818;
-  --ansi-red: #f8f8f8;
+  --ansi-red: #ff0000;
```

Comment:
`#ff0000` is too bright, should be `#ab4642`. As per the comment above, the colors come from [base16](https://base16.netlify.app/previews/base16-default-dark.html)

Reply:
Ah I see. Updated.